src/Kiota.Builder/SearchProviders/GitHub/GitHubClient/Models/AppPermissions_organization_custom_roles.cs

```diff
@@ -1,7 +1,9 @@
-﻿namespace Kiota.Builder.SearchProviders.GitHub.GitHubClient.Models;
-/// <summary>The level of permission to grant the access token for custom repository roles management. This property is in beta and is subject to change.</summary>
-public enum AppPermissions_organization_custom_roles
+﻿namespace Kiota.Builder.SearchProviders.GitHub.GitHubClient.Models
 {
-    Read,
-    Write,
+    /// <summary>The level of permission to grant the access token for custom repository roles management. This property is in beta and is subject to change.</summary>
+    public enum AppPermissions_organization_custom_roles
```

Comment:
Looking at this, I'm thinking maybe we can extend the work in https://github.com/microsoft/kiota/pull/2202 to also unmangle the class names?

Reply:
Resolving this. Looks like https://github.com/microsoft/kiota/pull/2261 has done some initial work we can leverage on.